components/brave_news/browser/resources/Sidebar.tsx

```diff
@@ -8,8 +8,9 @@ import * as React from 'react';
 import styled, { css } from 'styled-components';
```

Comment:
Sweet, mind if I do that in a separate PR? I've got some changes to this component coming up, so make it anchor to the feed, and on the left, so it might be easiest to do this there.

Reply:
https://github.com/brave/brave-core/pull/20980